### DISCUSSION TITLE
Conflict when using filament and CkEditor5 (v38.0.1)

### DISCUSSION THREAD

**Discussion by LucVK** [filamentphp/filament#6659] [Help]
I'm integrating CkEditor5 on a filament page and get a javascript error stating that 'hex-color-picker' (vanilla-colorful) is already used on the CustomElementRegistry.

> Uncaught DOMException: Failed to execute 'define' on 'CustomElementRegistry': the name "hex-color-picker" has already been used with this registry
    at 97 (http://laravel.rv:82/filament/assets/app.js?id=ec835eb3e622d56b7f9a6fdd6e61611e:27:413325)
    at i (http://laravel.rv:82/filament/assets/app.js?id=ec835eb3e622d56b7f9a6fdd6e61611e:81:141622)
    at http://laravel.rv:82/filament/assets/app.js?id=ec835eb3e622d56b7f9a6fdd6e61611e:81:142595
    at i.O (http://laravel.rv:82/filament/assets/app.js?id=ec835eb3e622d56b7f9a6fdd6e61611e:81:141888)
    at http://laravel.rv:82/filament/assets/app.js?id=ec835eb3e622d56b7f9a6fdd6e61611e:81:142644
    at http://laravel.rv:82/filament/assets/app.js?id=ec835eb3e622d56b7f9a6fdd6e61611e:81:142650

Indeed CkEditor and Filament Forms both use vanilla-colorful/hex-color-picker. 

I managed to work around the problem by using the following hack (and probably loosing the hex editor in filament forms):
1. Getting the latest source code of filament
2. Commenting out the use of hex-color-picker in 'packages/forms/resources/js/components/color-picker.js (see below)
3. Rebuilding the dist directory (npm run build)
4. Copying the new files in dist to the vendor/filament directory

> /* import 'vanilla-colorful/hex-color-picker' */
> import 'vanilla-colorful/hsl-string-color-picker'
> import 'vanilla-colorful/rgb-string-color-picker'
> import 'vanilla-colorful/rgba-string-color-picker'
> 
> import '../../css/components/color-picker.css'
> 
> export default (Alpine) => {
>     Alpine.data(
>         'colorPickerFormComponent',
> 

What is the correct why to solve this issue?

PS: this problem does not occur with previous versions of CkEditor, as they do not include hex-color-picker, but I cannot use them since I need the online builder which is only available for the latest version.

Br,
Luc

**Comment by JinO16** [comment]
I have encountered the same problem. How was it resolved
